assertj-core/src/main/java/org/assertj/core/error/ShouldBeInFloatingPointFormat.java

```diff
@@ -0,0 +1,15 @@
+package org.assertj.core.error;
+
+public class ShouldBeInFloatingPointFormat extends BasicErrorMessageFactory{
```

Comment:
Add a unit test for this, you can take ShouldBeEven_create_Test as an example to follow